access/ms-teams/msg_slice_test.go

```diff
@@ -0,0 +1,25 @@
+package main
```

Comment:
This file ends with `_test.go` but doesn't actually contain any tests

Reply:
Yes, but it contains the code essential for tests to work. `_test.go` indicates that this file belongs to the test domain.